src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/fee/FeeDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+
+public class FeeDto {
```

Comment:
Minor comment.  I personally prefer not having Dto on the end of domain object names.  But up to you if you want to keep.